sequel-jdbc-as400.gemspec

```diff
@@ -17,10 +17,11 @@ Gem::Specification.new do |s|
 
   s.add_runtime_dependency 'jdbc-jt400', '~> 9.1'
   s.add_runtime_dependency 'sequel', '~> 5.0'
+
```

Comment:
This file has the line `s.required_ruby_version = '>= 1.8.7'`, which is about which version is supported.